src/main/java/com/ibm/stocator/fs/ObjectStoreFileSystem.java

```diff
@@ -287,16 +288,16 @@ public boolean delete(Path f, boolean recursive) throws IOException {
   @Override
   public FileStatus[] listStatus(Path f) throws FileNotFoundException, IOException {
     LOG.debug("List status of {}", f.toString());
-    return listStatus(f, null);
+    return listStatus(f, DEFAULT_FILTER);
```

Comment:
This is because `public FileStatus[] listStatus(Path f, PathFilter filter, boolean prefixBased)` will eventually get called. The `DEFAULT_FILTER` is there in case there isn't a PathFilter that is passed. So we want to be able to handle cases when a filter is specified and when it isn't.